examples/examples.py

```diff
@@ -1,21 +1,22 @@
 from mainflux import sdk
+import json
 
 default_url = "http://localhost"
 
```

Comment:
You can assign a value's to the access_token and rest variables like this:

`access_token = "<access_token>"
refresh_token = "<refresh_token>"

email="<email>"
password ="<password>"`

and avoid it in every func

`"""To start working with the Mainflux system,
you need to create a user account"""
mf_resp = mfsdk.users.create(
    user={"credentials": {"identity": email, "secret": password}},
    token="",
)
if mf_resp.error.status == 0:
    print(mf_resp.value)
else:
    print(mf_resp.error.message)`

@dborovcanin and @drasko what do you think ?